components/gitpod-protocol/src/protocol.ts

```diff
@@ -567,7 +567,14 @@ export interface VSCodeConfig {
     extensions?: string[];
 }
 
+export interface RepositoryCloneInformation {
+    url: string;
+    checkoutLocation?: string;
```

Comment:
what happens when the checkoutLocation is undefined? Where do we clone the additional repo?

Reply:
It defaults to the simple repository name, symmetrically to how we do it with the main repository.
See 
https://github.com/gitpod-io/gitpod/pull/7785/files#diff-af5f60bd3ac1a9b1771300c23f780b8fa11d821496acd6d8358739a1d802cb16R166